src/client/systems/NetworkingClient.cpp

```diff
@@ -260,9 +260,34 @@ void NetworkingClient::unreliableSend() {
     }
 }
 
+void NetworkingClient::sendNatPunch() {
```

Comment:
the UDP sending logic here might be possible to dedup but it's kind of annoying since the packets have different arguments here and in the Input command send

Reply:
maybe the sending function will just take a packet that you want sent